passByReference

```diff
@@ -0,0 +1,29 @@
+number = [1, 19, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]
+print("before", number)  #before [1, 19, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]
+
+#remove odd numbers from list
+def findeven(number):
+    itr=0
+    while(itr!=len(number)):
```

Comment:
The core logic is okay but using a for loop would make the code look more precise
You won't need the else part.